### ISSUE TITLE
I Can Use Anything?

### ISSUE THREAD

**Issue by CellIJel** [Gegy/whats-that-slot#10]
for some reason when you see what can go in there it shows other things you can't normally use with that block. I don't think this is really a bug, but I feel like only things that you can use with that block should show up.

PS: All mods I'm using are client-side mods, if you need proof, I'm more than happy to provide you with It.
![2022-08-20_21 00 32](https://user-images.githubusercontent.com/111668702/185773575-d7904224-c907-4675-89c0-b44e29203bf3.png)

**Comment by Gegy** [comment]
I agree, it would be very nice if this would display specific items that are usable as a furnace recipe input! I also agree on the technicalities- this may be more of a feature than a bug, since you can technically put the items in those slots 😄 
It's a bit of a tricky case because all of these kinds of cases would have to be handled on a case-by-case basis, since it's impossible to know for modded content whether an item in a slot will produce an output (and it can of course vary based on other items, like in normal crafting)

But this would still be a nice touch for furnaces, at the very least! Especially given the different kinds of furnaces 🙂

**Comment by CellIJel** [comment]
that makes sense, maybe make it so it only shows items there if it makes an output, either way its still a cool mod!
